.gitmodules

```diff
@@ -24,7 +24,7 @@
 	url = git://github.com/opennetworklinux/ONLP-quanta
 [submodule "submodules/linux"]
 	path = submodules/linux
-	url = git://github.com/opennetworklinux/linux
+	url = git://github.com/InterfaceMasters/linux
```

Comment:
Hi Vitally,

This is not going to work, I assume if we merge your linux changes in, this will not be necessary.